op_vault.go

```diff
@@ -236,3 +238,19 @@ func getVaultSecret(secret string, subkey string) (string, error) {
 	DEBUG("  success.")
 	return v.(string), nil
 }
+
+func parsePath(path string) (secret, key string) {
+	secret = path
+	if len(path) >= 2 { //must contain at least "a:", so two characters
+		for i := len(path) - 1; i >= 0; i-- {
+			if path[i] == ':' {
+				secret = path[0:i]
+				key = path[i+1 : len(path)]
+				break
+			}
+		}
+	} else if len(path) == 1 && path[0] == ':' { // if just ":"
+		secret = ""
+	}
+	return
+}
```

Comment:
Would this be any cleaner by using strings.LastIndex(), and splitting at that index?